api-specifications/properties.json

```
@@ -70,6 +70,11 @@
             "type": "#/definitions/Action"
           }
         },
+        "runtimeExpressionEvaluation": {
+          "description": "Boolean to determine the parallel evaluation of expression. Set to true if you want the expression to be evaluated between the actions (instead of upfront).",
```

So with this off it all expressions are evaluated when you press the button, right? Then I would word this something like this
```suggestion
          "description": "Boolean to determine the timing for evaluating expressions. Set to true if you want the expression to be evaluated before each action (instead of evaluating all when button is pressed).",
```

---

I mean at least change this, upfront means something else
```suggestion
          "description": "Boolean to determine the parallel evaluation of expression. Set to true if you want the expression to be evaluated between the actions (instead of up front).",
```